mathml/relations/css-styling/dynamic-dir-1.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Test dynamically changing dir attribute</title></head>
+  <link rel="help" href="https://mathml-refresh.github.io/mathml-core/#cssproperties">
+  <meta name="assert" content="The dir attribute should update direction map to css properties dynamically">
+  <link rel="match" href="dynamic-dir-1-ref.html">
+  <script>
+      window.addEventListener("DOMContentLoaded", function() {
+        var x = document.getElementsByTagName("math");
+        x[0].setAttribute("dir", "rtl");
+        x[1].setAttribute("dir", "rtl");
+        x[2].removeAttribute("dir");
+
+        x = document.getElementsByTagName("mstyle");
+        x[0].setAttribute("dir", "rtl");
+        x[1].setAttribute("dir", "rtl");
+        x[2].removeAttribute("dir");
+    
+        x = document.getElementsByTagName("mrow");
+        x[0].setAttribute("dir", "rtl");
+        x[1].setAttribute("dir", "rtl");
+        x[2].removeAttribute("dir");
+      })
```

Comment:
This can be rewritten in one loop:

["math", "mstyle", "mrow"].forEach((tag) => {
  var x = document.getElementsByTagName(tag);
  ...
});

You can also add a code comment for each change:
- attaching new dir attribute 
- changing dir attribute value from "ltr" to "rtl"
- removing dir attribute

Reply:
Sure, this is a literal port of what is in the mozilla tests, I was trying to change as little as possible my first run through... Happy to change that